### PR TITLE
[Docker] Fix image 

### PR DESCRIPTION
**Pull request by Herklos**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -18,7 +21,6 @@ install:
   - python3 -m pip install --prefer-binary -r dev_requirements.txt -r requirements.txt
   - mkdir user
   - cp ./config/default_config.json ./user/config.json
-#  - python3 start.py -p install all
```
Comment (GuillaumeDSM) [inline .travis.yml:21]: should we remove it ?

File: .travis.yml [hunk 2]
```diff
@@ -104,3 +124,23 @@ jobs:
...
+    - name: "Linux - Heroku - Deploy"
+      stage: edge
+      os: linux
+      before_install: echo "$HEROKU_TOKEN" | docker login --username=_ --password-stdin registry.heroku.com
+      install: skip
+      script: skip
+      before_deploy:
+        - docker pull drakkarsoftware/octobot:$TRAVIS_BRANCH
+        - docker tag drakkarsoftware/octobot:$TRAVIS_BRANCH registry.heroku.com/$HEROKU_APP_NAME/octobot
+        - docker push registry.heroku.com/$HEROKU_APP_NAME/octobot
```
Comment (GuillaumeDSM) [inline .travis.yml:137]: 💯

File: .travis.yml [hunk 3]
```diff
@@ -104,3 +124,23 @@ jobs:
...
+      deploy:
+        - provider: script
+          script: heroku container:release octobot -a $HEROKU_APP_NAME
```
Comment (GuillaumeDSM) [inline .travis.yml:140]: 🎉

File: .travis.yml (hunk 1 above)
Comment (Herklos) [reply .travis.yml:21]: We should adapt it when we'll migrate the tests I think

**Pull request merged**